README.md

```diff
@@ -1,6 +1,10 @@
-ASP.NET Core MVC Precompilation
+ ASP.NET Core MVC Precompilation (obsolete)
 ===
 
+## Obsolete
```

Comment:
How about we remove "Obsolete" and replace the text with the following note?:

"NOTE: This repo is solely for maintenance of the existing MVC precompilation feature. Future work on Razor compilation is now being handled in the [Razor](https://github.com/aspnet/razor repo. See https://github.com/aspnet/Razor/issues/1740 for additional details."

Reply:
@Eilon, any objections? If not, I'll just use @danroth27's suggestion.

Reply:
Yeah this sounds good to me.